### ISSUE TITLE
failed to produce the expected output file

### ISSUE THREAD

**Issue by amo13** [unpaper/unpaper#108]
This prevents my instance of [Paperless-ngx](https://github.com/paperless-ngx/paperless-ngx) from consuming new documents. Paperless calls ocrmypdf which apparently in turn calls unpaper. A few people, myself included, [reported this first to ocrmypdf](https://github.com/ocrmypdf/OCRmyPDF/issues/926) but this could well be an issue with unpaper. At least, a person from ocrmypdf's issue tracker says so.
You can find some more info on the linked issue on the tracker of ocrmypdf, but here is my log:

```
Mar 11 15:19:51 archvm paperless-manage[86153]: 14:19:51 [Q] ERROR Failed [Meldebescheinigung.jpg] - Meldebescheinigung.jpg: Error while consuming document Meldebescheinigung.jpg: SubprocessOutputError: unpaper: failed to produce the expected output file.  Called with: ['unpaper', '-v', '--dpi', '72.0', '--layout', 'none', '--mask-scan-size', '100', '--no-border-align', '--no-mask-center', '--no-grayfilter', '--no-blackfilter', '--no-deskew', '/tmp/tmp66wznyew/input.pnm', '/tmp/tmp66wznyew/output.ppm'] : Traceback (most recent call last):
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/lib/python3.10/site-packages/ocrmypdf/_exec/unpaper.py", line 122, in run_unpaper
Mar 11 15:19:51 archvm paperless-manage[86153]:     with Image.open(output_pnm) as imout:
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/lib/python3.10/site-packages/PIL/Image.py", line 3008, in open
Mar 11 15:19:51 archvm paperless-manage[86153]:     raise UnidentifiedImageError(
Mar 11 15:19:51 archvm paperless-manage[86153]: PIL.UnidentifiedImageError: cannot identify image file '/tmp/tmp66wznyew/output.ppm'
Mar 11 15:19:51 archvm paperless-manage[86153]: The above exception was the direct cause of the following exception:
Mar 11 15:19:51 archvm paperless-manage[86153]: Traceback (most recent call last):
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/share/paperless/src/paperless_tesseract/parsers.py", line 241, in parse
Mar 11 15:19:51 archvm paperless-manage[86153]:     ocrmypdf.ocr(**args)
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/lib/python3.10/site-packages/ocrmypdf/api.py", line 337, in ocr
Mar 11 15:19:51 archvm paperless-manage[86153]:     return run_pipeline(options=options, plugin_manager=plugin_manager, api=True)
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/lib/python3.10/site-packages/ocrmypdf/_sync.py", line 385, in run_pipeline
Mar 11 15:19:51 archvm paperless-manage[86153]:     exec_concurrent(context, executor)
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/lib/python3.10/site-packages/ocrmypdf/_sync.py", line 274, in exec_concurrent
Mar 11 15:19:51 archvm paperless-manage[86153]:     executor(
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/lib/python3.10/site-packages/ocrmypdf/_concurrent.py", line 82, in __call__
Mar 11 15:19:51 archvm paperless-manage[86153]:     self._execute(
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/lib/python3.10/site-packages/ocrmypdf/builtin_plugins/concurrency.py", line 135, in _execute
Mar 11 15:19:51 archvm paperless-manage[86153]:     result = future.result()
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/lib/python3.10/concurrent/futures/_base.py", line 439, in result
Mar 11 15:19:51 archvm paperless-manage[86153]:     return self.__get_result()
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/lib/python3.10/concurrent/futures/_base.py", line 391, in __get_result
Mar 11 15:19:51 archvm paperless-manage[86153]:     raise self._exception
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/lib/python3.10/concurrent/futures/thread.py", line 58, in run
Mar 11 15:19:51 archvm paperless-manage[86153]:     result = self.fn(*self.args, **self.kwargs)
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/lib/python3.10/site-packages/ocrmypdf/_sync.py", line 192, in exec_page_sync
Mar 11 15:19:51 archvm paperless-manage[86153]:     ocr_image, preprocess_out = make_intermediate_images(
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/lib/python3.10/site-packages/ocrmypdf/_sync.py", line 161, in make_intermediate_images
Mar 11 15:19:51 archvm paperless-manage[86153]:     ocr_image = preprocess(
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/lib/python3.10/site-packages/ocrmypdf/_sync.py", line 108, in preprocess
Mar 11 15:19:51 archvm paperless-manage[86153]:     image = preprocess_clean(image, page_context)
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/lib/python3.10/site-packages/ocrmypdf/_pipeline.py", line 497, in preprocess_clean
Mar 11 15:19:51 archvm paperless-manage[86153]:     return unpaper.clean(
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/lib/python3.10/site-packages/ocrmypdf/_exec/unpaper.py", line 160, in clean
Mar 11 15:19:51 archvm paperless-manage[86153]:     run_unpaper(input_file, output_file, dpi=dpi, mode_args=unpaper_args)
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/lib/python3.10/site-packages/ocrmypdf/_exec/unpaper.py", line 125, in run_unpaper
Mar 11 15:19:51 archvm paperless-manage[86153]:     raise SubprocessOutputError(
Mar 11 15:19:51 archvm paperless-manage[86153]: ocrmypdf.exceptions.SubprocessOutputError: unpaper: failed to produce the expected output file.  Called with: ['unpaper', '-v', '--dpi', '72.0', '--layout', 'none', '--mask-scan-size', '100', '--no-border-align', '--no-mask-center', '--no-grayfilter', '--no-blackfilter', '--no-deskew', '/tmp/tmp66wznyew/input.pnm', '/tmp/tmp66wznyew/output.ppm']
Mar 11 15:19:51 archvm paperless-manage[86153]: During handling of the above exception, another exception occurred:
Mar 11 15:19:51 archvm paperless-manage[86153]: Traceback (most recent call last):
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/lib/python3.10/site-packages/asgiref/sync.py", line 266, in main_wrap
Mar 11 15:19:51 archvm paperless-manage[86153]:     raise exc_info[1]
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/share/paperless/src/documents/consumer.py", line 248, in try_consume_file
Mar 11 15:19:51 archvm paperless-manage[86153]:     document_parser.parse(self.path, mime_type, self.filename)
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/share/paperless/src/paperless_tesseract/parsers.py", line 290, in parse
Mar 11 15:19:51 archvm paperless-manage[86153]:     raise ParseError(f"{e.__class__.__name__}: {str(e)}")
Mar 11 15:19:51 archvm paperless-manage[86153]: documents.parsers.ParseError: SubprocessOutputError: unpaper: failed to produce the expected output file.  Called with: ['unpaper', '-v', '--dpi', '72.0', '--layout', 'none', '--mask-scan-size', '100', '--no-border-align', '--no-mask-center', '--no-grayfilter', '--no-blackfilter', '--no-deskew', '/tmp/tmp66wznyew/input.pnm', '/tmp/tmp66wznyew/output.ppm']
Mar 11 15:19:51 archvm paperless-manage[86153]: During handling of the above exception, another exception occurred:
Mar 11 15:19:51 archvm paperless-manage[86153]: Traceback (most recent call last):
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/lib/python3.10/site-packages/django_q/cluster.py", line 432, in worker
Mar 11 15:19:51 archvm paperless-manage[86153]:     res = f(*task["args"], **task["kwargs"])
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/share/paperless/src/documents/tasks.py", line 74, in consume_file
Mar 11 15:19:51 archvm paperless-manage[86153]:     document = Consumer().try_consume_file(
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/share/paperless/src/documents/consumer.py", line 266, in try_consume_file
Mar 11 15:19:51 archvm paperless-manage[86153]:     self._fail(
Mar 11 15:19:51 archvm paperless-manage[86153]:   File "/usr/share/paperless/src/documents/consumer.py", line 70, in _fail
Mar 11 15:19:51 archvm paperless-manage[86153]:     raise ConsumerError(f"{self.filename}: {log_message or message}")
Mar 11 15:19:51 archvm paperless-manage[86153]: documents.consumer.ConsumerError: Meldebescheinigung.jpg: Error while consuming document Meldebescheinigung.jpg: SubprocessOutputError: unpaper: failed to produce the expected output file.  Called with: ['unpaper', '-v', '--dpi', '72.0', '--layout', 'none', '--mask-scan-size', '100', '--no-border-align', '--no-mask-center', '--no-grayfilter', '--no-blackfilter', '--no-deskew', '/tmp/tmp66wznyew/input.pnm', '/tmp/tmp66wznyew/output.ppm']
```
I use archlinux and unpaper 6.1 (latest in repo).

**Comment by LarsKumbier** [comment]
This behavior only affected arch-based distros and has been fixed in Arch's unpaper version 6.1-9. Manjaro has not updated it yet. If you need a quickfix, you can download it from the [arch package page](https://archlinux.org/packages/community/x86_64/unpaper/download) and install it via `pamac -U unpaper-*.zst`

**Comment by amo13** [comment]
You are absolutely right and you were faster than me to notice the update in the repository. Thank you!
